tests/test_big_modeling.py

```diff
@@ -40,16 +40,24 @@
     require_multi_device,
     require_multi_gpu,
     require_non_cpu,
+    require_non_hpu,
     require_non_torch_xla,
     slow,
     torch_device,
 )
-from accelerate.utils import offload_state_dict
+from accelerate.utils import is_hpu_available, offload_state_dict
 
 
 logger = logging.getLogger(__name__)
 torch_device = f"{torch_device}:0" if torch_device != "cpu" else "cpu"
 
+if torch_device.startswith("hpu"):
+    ATOL = 1e-3
+    RTOL = 1e-3
+else:
+    ATOL = 1e-5
+    RTOL = 1e-5
```

Comment:
the big modeling tests compare computations done on cpu vs accelerator, resulting in lots of numerical differences.